### PR TITLE
#791 Fixes command bar button layout shifting on hover in HighContrast mode

### PR DESCRIPTION
**Pull request by aleksandrjPersonal**
#### Pull request checklist

- [ ] Addresses an existing issue: #0000
- [ ] Include a change request file if publishing 
- [ ] New feature, bugfix, or enhancement
  - [ ] Includes tests
- [ ] Documentation update

#### Description of changes

(give an overview)

#### Focus areas to test

(optional)

### CODE REVIEW

**Comment by aleksandrjPersonal** [comment]
adding rush change

File: common/changes/791_CommandBarHCShift_2017-03-01-22-42.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "adds pseudo:after class to highContrastBorder mixin, renders border on hover",
```
Comment (mikewheaton) [inline common/changes/791_CommandBarHCShift_2017-03-01-22-42.json]: This comment shows up in the release notes, so we usually start it with the component name and then a brief description of the functionality. E.g. "CommandBar: Improves accessibility by making borders visible in high contrast mode"

**Comment by dzearing** [comment]
Did not fill out PR form, or have useful title. Please do this.

File: common/changes/791_CommandBarHCShift_2017-03-01-22-42.json (hunk 1 above)
Comment (aleksandrjPersonal) [inline common/changes/791_CommandBarHCShift_2017-03-01-22-42.json]: Change file updated

**Pull request merged**